test/test_SimulatedChopper.py

```diff
@@ -19,8 +19,8 @@
 
 import unittest
 
-from lewis.devices.chopper.states import DefaultIdleState
-from lewis.devices.chopper import SimulatedChopper
+from lewis.devices.chopper.devices.states import DefaultIdleState
+from lewis.devices.chopper.devices.device import SimulatedChopper
```

Comment:
It does look weird now, but I'm not sure how to resolve it. What should we call it instead?

If we call it `chopper.py`, we just wind up with:
```python
from lewis.devices.chopper.devices.chopper import ...
```
Which isn't exactly an improvement.

Keep in mind though this is a structure change in preparation for more changes to come (which may fix this awkward naming):
- Devices should soon be moved to their own repository
- Tests should soon be moved to a `tests/` directory alongside `devices/` and `interfaces/`

I think the import statements will become clearer as we make those changes.